.vscode/settings.json

```
@@ -29,5 +29,23 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true
   },
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "json.schemas": [
+    {
+      "fileMatch": ["**/infoAssessment.json"],
```

Are `{...}` groups supported here? If so, it'd be nice to be as specific as possible, ditto for the others.

```suggestion
      "fileMatch": ["{exampleCourse,testCourse,templateCourse}/courseInstances/**/assessments/**/infoAssessment.json"],
```

---

Resolved in d751b45.